packages/vm/lib/evm/interpreter.ts

```diff
@@ -195,12 +195,12 @@ export default class Interpreter {
      * @property {String} opcode the next opcode to be ran
      * @property {BN} gasLeft amount of gasLeft
      * @property {Array} stack an `Array` of `Buffers` containing the stack
-     * @property {Account} account the [`Account`](https://github.com/ethereum/ethereumjs-account) which owns the code running
+     * @property {Account} account the Account which owns the code running
      * @property {Buffer} address the address of the `account`
      * @property {Number} depth the current number of calls deep the contract is
      * @property {Buffer} memory the memory of the VM as a `buffer`
      * @property {BN} memoryWordCount current size of memory in words
-     * @property {StateManager} stateManager a [`StateManager`](stateManager.md) instance (Beta API)
+     * @property {StateManager} stateManager a [[StateManager]] instance
```

Comment:
Had to look up these [symbol references](https://typedoc.org/guides/doccomments/#symbol-references) (double square brackets) - so linking to internal classes, members or functions, really nice functionality! 😄  Just dropping here for others as a reference.